backend/routes.js

```diff
@@ -229,6 +230,8 @@ const setup = ({
     amendCaseNNoteRouter({ prisonApi, caseNotesApi, logError })
   )
 
+  router.get(['/content', '/content/:path'], contentController({ logError }))
```

Comment:
I think there's a route setup in the setupReactRoutes file as well. It doesn't matter if not deleted though.

Reply:
Deleted and pushed 👍  Cheers